openedx/core/djangoapps/credit/signature.py

```diff
@@ -54,10 +54,10 @@ def signature(params, shared_secret):
         str: The 32-character signature.
 
     """
-    encoded_params = "".join([
-        "{key}:{value}".format(key=key, value=params[key])
+    encoded_params = u"".join([
```

Comment:
This needs unit tests to verify that Unicode characters are now handled correctly (and that failed before the fixes)